### DISCUSSION TITLE
Can I disable the Quarto Preview button for non-.qmd files?

### DISCUSSION THREAD

**Discussion by cbrnr** [quarto-dev/quarto-cli#8928] [Q&A] [answered]
### Description

The Quarto VS Code extension includes a Preview toolbar button, which also shows up for regular Markdown (`.md`) files. However, VS Code already ships a built-in Markdown preview (the icon with the magnifying glass), so I'd rather disable/hide the Quarto preview button for all files that are not `.qmd` files. Is this possible?

**Comment by mcanouil** [comment, marked as answer]
VSCode extension repository is <https://github.com/quarto-dev/quarto>.

1. Check Quarto VSCode settings
2. Check for already open issues on <https://github.com/quarto-dev/quarto>
3. Open a new issue to make a feature request.

**Reply by cbrnr** [reply]
Sorry, I didn't realize this forum is only for the CLI. I've opened a new issue here: https://github.com/quarto-dev/quarto/issues/388

**Reply by mcanouil** [reply]
No apology needed.
This repository is indeed for discussions about Quarto in general.

But bug reports and feature requests regarding Visual editor and the VSCode extension are in <https://github.com/quarto-dev/quarto>.

**Reply by mcanouil** [reply]
@cbrnr May I suggest you to add a screenshot to your issue?
